python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -111,14 +118,18 @@ def write_catalog(self, obsHistID, out_dir='.', fov=2):
                     os.path.join(out_dir, bright_star_name): bright_cat}
         parallelCatalogWriter(cat_dict, chunk_size=100000, write_header=False)
 
-        cat = self.instcats.DESCQACat(bulgeDESCQAObject(self.descqa_catalog),
-                                      obs_metadata=obs_md,
+        bulge_db = bulgeDESCQAObject(self.descqa_catalog)
```

Comment:
this should be `bulgeDESCQAObject_protoDC2`

Reply:
I rename those classes at the import statement on lines 22-23.  Since this is sort of a kluge for protoDC2, I wanted to localize where these classes are used, thinking that for future runs, we won't use them.

Reply:
Oh.  I see.  Sorry.  Merge away.

Reply:
np. Thanks for looking this over!